backend/app/views/spree/admin/stock_transfers/_stock_transfer_table.html.erb

```diff
@@ -0,0 +1,46 @@
+<% if @stock_transfers.any? %>
+  <%= render 'search' if @stock_transfers.size > 9 || params[:q].present? %>
```

Comment:
I'm not sure about this one. I see the rain behind, but we don't have this kind of behavior in any other table we have. I think this should be discussed as an overall UI improvement on a separate issue/PR